class-vip-support-user.php

```diff
@@ -436,8 +436,10 @@ public function action_password_reset( $user ) {
 			return;
 		}
 
-		$this->mark_user_email_verified( $user->ID, $user->user_email );
-		$this->promote_user_to_vip_support( $user->ID );
+		if ( is_proxied_automattician() ) {
+			$this->mark_user_email_verified( $user->ID, $user->user_email );
```

Comment:
I don't think this code will run, because for `is_proxied_automattician()` to return `true`, `is_automattician()` must return `true`, which requires a verified email in the first place, which _should_ get reset with a password reset (right?).